src/doc/trpl/traits.md

```diff
@@ -19,7 +19,8 @@ impl Circle {
 
 [methodsyntax]: method-syntax.html
 
-Traits are similar, except that we define a trait with just the method
+[Traits](../reference.html#traits) are similar,
```

Comment:
We don't generally link to the reference from the book, or at least, not that I'm aware of. So, while I don't think this is wrong, but it is a bit unusual.